Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/NotificationReceiver.kt

```diff
@@ -0,0 +1,31 @@
+package de.rki.coronawarnapp.notification
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import dagger.android.AndroidInjection
+import de.rki.coronawarnapp.notification.NotificationConstants.NOTIFICATION_ID
+import de.rki.coronawarnapp.notification.NotificationConstants.POSITIVE_RESULT_NOTIFICATION_ID
+import timber.log.Timber
+import javax.inject.Inject
+
+typealias NotificationId = Int
+
+class NotificationReceiver : BroadcastReceiver() {
+
+    @Inject lateinit var testResultNotificationService: TestResultNotificationService
+
+    override fun onReceive(context: Context, intent: Intent) {
+        AndroidInjection.inject(this, context)
+        when (val notificationId = intent.getIntExtra(NOTIFICATION_ID, Int.MIN_VALUE)) {
+            POSITIVE_RESULT_NOTIFICATION_ID ->
+                testResultNotificationService.showPositiveTestResultNotification(notificationId)
```

Comment:
Maybe also add a log here, to be sure that it was executed, otherwise we only see the `else` path.

Reply:
The NotificationHelper logs all built notifications but will add it anyway in case it gets lost somewhere in between